backend/resources/akvo/lumen/migrations/tenants/012-plans.up.sql

```diff
@@ -0,0 +1,118 @@
+CREATE TABLE tier (
+     id text PRIMARY KEY DEFAULT gen_random_uuid()::text,
+     title text NOT NULL UNIQUE CHECK (title IN ('lite','standard','pro','enterprise'))
+);
+
+INSERT INTO tier (title) VALUES ('lite');
+INSERT INTO tier (title) VALUES ('standard');
+INSERT INTO tier (title) VALUES ('pro');
+INSERT INTO tier (title) VALUES ('enterprise');
+-- ;;
+
+CREATE OR REPLACE FUNCTION standard_tier_id() RETURNS text LANGUAGE SQL AS
+$$ SELECT id FROM tier WHERE title = 'standard'; $$;
+
+CREATE TABLE plan (
+     id text PRIMARY KEY DEFAULT gen_random_uuid()::text,
+     tier text REFERENCES tier (id) DEFAULT standard_tier_id(),
+     created timestamptz DEFAULT now(),
+     ends timestamptz DEFAULT 'infinity',
+     author jsonb
+);
+-- ;;
+
+CREATE OR REPLACE FUNCTION public.end_plans_fn ()
+RETURNS trigger AS
+$$
+BEGIN
+     UPDATE plan SET ends = now() WHERE ends = 'infinity';
```

Comment:
`plan` is a table with only ever one row, right? Which is why you don't need a `WHERE id=...`? Or is the row with `infinity` always the "current" plan?